contents/articles/2018-01_pupil_community/index.md

```diff
@@ -0,0 +1,64 @@
+--- 
+title: Pupil Community
+date: Fri Jan 12 2018 14:08:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Pupil is a community driven development project. Check out what the community has been doing with Pupil..."
+featured_img: ""
+featured_img_thumb: ""
+---
+
+Happy 2018!
+Our first blog post of the year will be about the Pupil community, looking at how much Pupil has grown!
+Pupil is a community driven development project to make eye tracking hardware that is accessible, hackable, affordable and software that is open source.
+
+Check out what the community has been doing! Here are a list of custom tools and projects built with Pupil.
+
+## Projects
+
+### OptiTrack and Pupil Labs Python Recorder
+https://github.com/mdfeist/OptiTrack-and-Pupil-Labs-Python-Recorder
+
+- A simple python script that can record data from Pupil Labs and OptiTrack
+
+### Graph viz eye tracker
+https://github.com/Saftophobia/graph-viz-eye-tracker
+
+- Creating a gaze-aided graph navigating application using Unity3D and Pupil-lab.
+
+### gcvlc - Gaze-controlled VLC Player, Pupil Labs Plugin
+https://github.com/MPIK-COMMS/gcvlc
+
+- Gaze-controlled VLC player used with the Pupil Labs Capture software.
+
+### Offline Gaze Mapping Tools for Pupil Labs Glasses
+https://github.com/jeffmacinnes/pl_gazeMapping_offline
+
+- Offline Gaze Mapping Tools for Pupil Labs Glasses
+
+### Pupil Labs usybus Controller
+https://github.com/Lahorde/pupil-labs-usybus-controller
+
+- This plugin exports pupillabs eye tracking gaze to Ivybus
+
+### pupil-progs 
+https://github.com/SGBon/pupil-progs
+
+- Small Programs to use with the Eye Tracker by Pupil Labs
+
+### Eye Tracking Research Data Analysis 
+https://github.com/qalhata/Eye_Tracking_Research_Data_Analysis
+
+- Python Scripts to analyse Pupillometrics captured using the Pupil Labs Eye tracking system
+
+### ODST - Optical Dimension Spatial Thinking
+https://github.com/Ruzzy77/ODST
+
+- Computer Controlling System using Eye-Tracking and Voice Recognition
+
+### Lab Matlab Control 
+https://github.com/TheGoldLab/Lab-Matlab-Control/tree/62d56585ef2fda3c17045dfcdc69e159eb317a38
+
+- Custom utilities for experimental control for the Gold Lab
+
+
+Our Open source software is free for you to use and explore. Make a suggestion, fix a bug, improve documentation, or contribute code to a project. Get involved and contribute and be part of Pupil. Submit your project to the github repository by creating a PR to the list.
```

Comment:
what does this mean? What repo? Link?

Reply:
Me and @willpatera were discussing making a blog post about the tools and projects the community has done. We thought of making a new repo that is a list of all the projects we've found and that other people who has developed something with pupil can add their project to the list via PRs.

Reply:
sounds good. I understand now that this is in WIP mode. A repo with one page and PR contributions sounds good!

Reply:
@mkassner I was considering making a new repository that has only a Readme file in the repository where community members can submit a PR to include their project, plugin, or script. We can use this as a resource to showcase community contributed work and as a resource for other members in the community who either want to find examples, starting points, or other people to work with. Let me know what you think about this. 

Possioble repo names:
`pupil-community`
`pupil-community-contributions`